src/main/java/com/redhat/devtools/lsp4ij/server/definition/launching/ClientConfigurationSettings.java

```diff
@@ -137,11 +160,21 @@ public static class ClientConfigurationWorkspaceSymbolSettings {
      */
     public boolean caseSensitive = false;
 
+    /**
+     * The statement terminator characters in the language grammar. Defaults to none.
+     */
+    public @NotNull String statementTerminatorCharacters = "";
```

Comment:
And these are the language grammar-level statement terminator characters for the language (if it even supports statement terminators).